torch/jit/__init__.py

```diff
@@ -2082,9 +2040,14 @@ def register_all(mod):
 
     for builtin, aten_op in builtin_ops:
         _builtin_table[id(builtin)] = aten_op
-    if not PY2:
-        _builtin_table[id(math.gcd)] = "aten::gcd"
-        _builtin_table[id(math.isfinite)] = "aten::isfinite"
+    math_ops = [x for x in dir(math) if callable(getattr(math, x))]
+    special_math_ops = ["remainder"]    # We bind this to aten::mathremainder
+    for op in math_ops:
```

Comment:
Is this sustainable? IIUC say if python3.8 add `math.xxx` in `math` module, this might fail?
This also implies number of `math` ops we support in torchscript varies based on the python version it's compiled with right? e.g. `math.xxx` is available in python2 but not python3, then it's only registered in torchscript in the python2 build. 
cc: @driazati is this behavior expected?